src/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: #000;
+}
+
+.calendar {
+  @extend %display-set;
+  height: $calendar-height;
+  @for $day-number from 1 to $day-count {
+    &--month-length_#{$day-number} {
+      .calendar__day:nth-child(n + #{$day-number + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  @each $day, $shift in $week-days {
+    &--start-day_#{$day} {
+      .calendar__day:nth-child(1) {
+        margin-left: $shift;
+      }
+    }
+  }
+
+  &__mockup {
+    display: flex;
+    justify-content: flex-start;
+    align-content: center;
+    flex-wrap: wrap;
+    padding: $mockup-padding;
+    gap: $gap-size;
+    min-width: $mockup-size;
+    max-width: $mockup-size;
+  }
+
+  &__day {
+    @extend %display-set;
+    align-items: center;
```

Comment:
To understand this code I needed to scroll down, find what is `%display-set` (it was not obvious from the name)
So here I would prefer to see exact rules but not such an extend

Reply:
Agree with suggestion. Usability of code is better with direct setting of display set. 
@Extend was add to the code for purpose of learning, to utilize possbilities of sass as much as possible.